api/api.py

```diff
@@ -146,12 +146,14 @@ async def get_gazettes(
     since: Optional[date] = Query(
         None,
         title="Since date",
-        description="Look for gazettes where the date is greater or equal than given date",
+        description="Year-Month-Day. Look for gazettes where the date is greater or equal than given date ", 
```

Comment:
I forgot to mention. Sorry... define the format like YYYY-MM-DD. This is necessary because we can define years like 2021 or 21, a day like 1 or 01 and so on...

Reply:
got it